chrome/renderer/BUILD.gn

```diff
@@ -112,6 +112,8 @@ static_library("renderer") {
     "worker_content_settings_client.h",
   ]
 
+  include_dirs = [ "//third_party/wrt/ewk/efl_integration" ]
+
```

Comment:
I checked that "include_dirs" are not inherited even if a build.gn file adds the gn file including that "include_dirs" to "deps" list by the build error below. So instead I will use the full path for those includes.

../../chrome/renderer/chrome_content_renderer_client.cc:155:10: fatal error: 'wrt/wrtwidget.h' file not found
#include "wrt/wrtwidget.h"

In file included from ../../chrome/renderer/chrome_content_renderer_client.cc:155:
../../third_party/wrt/ewk/efl_integration/wrt/wrtwidget.h:10:10: fatal error: 'common/content_switches_efl.h' file not found
#include "common/content_switches_efl.h"

In file included from ../../chrome/renderer/chrome_content_renderer_client.cc:155:
../../third_party/wrt/ewk/efl_integration/wrt/wrtwidget.h:13:10: fatal error: 'wrt/v8widget.h' file not found
#include "wrt/v8widget.h"